packages/front-end/components/Features/ConditionDisplay/index.tsx

```diff
@@ -104,14 +105,15 @@ export function MultiValuesDisplay({ values }: { values: string[] }) {
       {values.length > MULTI_VALUE_LIMIT && (
         <Tooltip
           body={
-            <div>
+            <div className="z-10">
               {values.slice(MULTI_VALUE_LIMIT).map((v, i) => (
-                <span key={i} className="mr-1 border px-2 bg-light rounded">
+                <span key={i} className={`${styles.Tooltip} ml-1`}>
                   {v}
                 </span>
               ))}
             </div>
           }
+          usePortal
```

Comment:
To fix the z-index layer issue I found on the Features page:
![image](https://github.com/user-attachments/assets/f888b99a-fab5-48f7-adcd-66869cd00fe9)